recipes-openxt/openxt/openxt-measuredlaunch/seal-system

```diff
@@ -225,6 +231,10 @@ forward)
         pcr_params="${pcr_params} -p 17 -p 18 -p 19"
     fi
 
+    if ! contains_only "${pcr8}" 0; then
+        umount /tmp/esp
```

Comment:
You already have a umount here: https://github.com/OpenXT/xenclient-oe/pull/890/commits/6f54bec36f0d6a4864ff94d3b229656e0af726a1#diff-fd5fec9e92a78f77ee7f1b54e95855deR189 within the if block that mounts it.

Reply:
Good catch, that other unmount needs to be removed. Thanks!